pipelines/respondent-home-ui.yml

```diff
@@ -164,9 +173,21 @@ jobs:
       command: map-route
       app_name: respondent-home-ui-preprod
       domain: ohs-alpha.onsdigital.uk
+  - task: smoke-tests
```

Comment:
Out of curiosity, why are these not run on `latest`?

Reply:
No reason but I'll it makes sense to add it

Reply:
Done

Reply:
Other changes are on this beast of a PR https://github.com/ONSdigital/respondent-home-ui/pull/22